src/Events/Custom_Tables/V1/Migration/Reports/Event_Report.php

```diff
@@ -526,11 +526,10 @@ public function __get( $prop ) {
 	 * The JSON serializer logic.
 	 *
 	 * @since 6.0.0
-	 *
-	 * @return mixed
 	 */
+	#[\ReturnTypeWillChange]
```

Comment:
just adding `: mixed` to the signature breaks some of our tests, so I patched it this way. Prevents deprecation notices on PHP 8.1